lib/logdna/client.rb

```diff
@@ -50,7 +50,6 @@ def schedule_flush
         flush if @flush_scheduled
       }
       thread = Thread.new { start_timer.call }
```

Comment:
I don't see any reasons for not going with the change. But could you please remove the "thread" variable since it is not used anymore? 
Thank you.

Reply:
👍